requirements.txt

```diff
@@ -81,10 +81,6 @@ docutils==0.18.1
     #   sphinxcontrib-bibtex
 entrypoints==0.4
     # via jupyter-client
-exceptiongroup==1.1.1
```

Comment:
No replacement code here, is that right?

Reply:
Correct!  I think this is no longer needed since the pinned dependencies are for Python 3.11.